jest.config.json

```diff
@@ -11,6 +11,7 @@
     "js",
     "jsx"
   ],
+  "testEnvironment": "node",
```

Comment:
this is required so that JSDom doesn't try to interfere with the global `URL` supplied by node - there are some related test refactors that had to happen alongside this in order to fix tests that touch browser-specific functions